templates/cluster-template.yaml

```diff
@@ -1,62 +1,73 @@
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
 metadata:
-  name: ${CLUSTER_NAME}
+  name: "${CLUSTER_NAME}"
 spec:
   clusterNetwork:
-    services:
-      cidrBlocks: ["${VIRTINK_SERVICE_CIDR}"]
     pods:
-      cidrBlocks: ["${VIRTINK_POD_NETWORK_CIDR}"]
-  controlPlaneRef:
-    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
-    kind: KubeadmControlPlane
-    name: ${CLUSTER_NAME}-cp
+      cidrBlocks: ["192.168.0.0/16"]
   infrastructureRef:
     apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
     kind: VirtinkCluster
-    name: ${CLUSTER_NAME}
+    name: "${CLUSTER_NAME}"
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    kind: KubeadmControlPlane
+    name: "${CLUSTER_NAME}-cp"
 ---
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: VirtinkCluster
 metadata:
-  name: ${CLUSTER_NAME}
-spec: {}
+  name: "${CLUSTER_NAME}"
+spec:
```

Comment:
LoadBalancer service type is required here, and it's external IP should be reachable in management cluster.

Reply:
The NodePort may not work in some cloud environments. The infrastructure cluster API server address may be a VIP, and only DNAT 6443 to actual infrastructure node.